_pages/contact.md

```diff
@@ -13,6 +13,6 @@ permalink: /contact/
 
 <div class="bg-white">
   <div class="container cntnr-wide px2 pt4 pb5">
-    For information or questions please contact us at <a href="mailto:login-contact@gsa.gov?subject=login.gov">login-contact@gsa.gov</a>.
+   First, check out answers to <a href="https://login.gov/help/">common questions</a>. Still have a question or a problem with the platform? Contact us at <a href="mailto:login-contact@gsa.gov?subject=login.gov">login-contact@gsa.gov</a>.
```

Comment:
1. small nit: looks like the indentation was changed here (should be indented by 2 spaces under the line below it)

2. Problem: our tests that work to make sure our links are consistent caught this link URL as inconsistent, let's make it:

    ```
    href="{{ '/help/' | relative_url }}"
    ```

    please! (this will mean the links still keep working on branch previews, etc etc)